kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -659,10 +662,18 @@ private void tagFile(String encodedPartition, String s3ObjectPath) {
       log.info("Tagged S3 object {} with starting offset {}, ending offset {}, record count {}",
           s3ObjectPath, startOffset, endOffset, recordCount);
     } catch (SdkClientException e) {
-      log.warn("Unable to tag S3 object {}. Ignoring.", s3ObjectPath, e);
+      if (ignoreTaggingErrors) {
+        log.warn("Unable to tag S3 object {}. Ignoring.", s3ObjectPath, e);
```

Comment:
Another way to avoid consuming errors would be to adjust the retry policy, passing the behaviour on tagging config and deciding there whether to retry.

Reply:
+1 , we could use - https://github.com/confluentinc/kafka-connect-storage-cloud/blob/21da6c93b7108a26060d55895b7261f48246f990/kafka-connect-s3/src/main/java/io/confluent/connect/s3/util/S3ErrorUtils.java#L15-L86

Reply:
this was just a mention.